src/Microsoft.ML.Transforms/Microsoft.ML.Transforms.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.ML</IncludeInPackage>
     <DefineConstants>CORECLR</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
>AllowUnsafeBlocks [](start = 5, length = 17)

just for one GEMV call... that's annoying.

Reply:
actually nevermind, it's going into HalLearners anyway

---
In reply to: [233679433](https://github.com/dotnet/machinelearning/pull/1580#discussion_r233679433) [](ancestors = 233679433)